### PR TITLE
Lint remaining services

### PR DESCRIPTION
**Pull request by elland**
And that's the lest of /services. Next, tools and libs!

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.

### CODE REVIEW

File: services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs [hunk 1]
```diff
@@ -186,11 +186,14 @@ sink = go
...
-                  runSpar $
-                    write insert (params LocalQuorum (tid, extid, uid))
+              askMigEnv
+                >>= ( \case
+                        DryRun -> pure ()
+                        NoDryRun ->
+                          runSpar $
+                            write insert (params LocalQuorum (tid, extid, uid))
+                    )
+                  . dryRun
```
Comment (fisx) [inline services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs:196]: I strongly disagree.  what does hlint say if we put brackets around `dryRun <$> askMigEnv` in the original code?

**Review by fisx — APPROVED** [review]
see comment, i'd like to see this changed into something not more obfuscating, but less.  but i'll leave it to the author to second-guess me.

File: services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs (hunk 1 above)
Comment (elland) [reply services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs:196]: Yeah, I'm not sold on this either… A few suggestions seem to be counter-productive when we have code like this.

Comment (elland) [reply services/spar/migrate-data/src/Spar/DataMigration/V1_ExternalIds.hs:196]: Same, redundant `<$>`. How about extracting the case match?

**Pull request merged**